sklearn/feature_selection/tests/test_mutual_info.py

```diff
@@ -252,3 +252,15 @@ def test_mutual_info_regression_X_int_dtype(global_random_seed):
     expected = mutual_info_regression(X_float, y, random_state=global_random_seed)
     result = mutual_info_regression(X, y, random_state=global_random_seed)
     assert_allclose(result, expected)
+
+
+def test_mutual_info_regression_n_jobs(global_random_seed):
```

Comment:
This looks good but we need to test both regression and classification.

One way would be the parametrize this function with the `mutual_info_regression` and `mutual_info_classification` and discretize the target for the second case.

Reply:
I have added another test. I am not familiar with test parametrization, so if what I have done is not enough, let me know and I will try to find references to it.